pkg/cmd/docgen/diagrams.go

```diff
@@ -462,11 +462,17 @@ var specs = []stmtSpec{
 		name:   "create_changefeed_stmt",
 		inline: []string{"changefeed_targets", "single_table_pattern_list", "opt_changefeed_sink", "opt_with_options", "kv_option_list", "kv_option"},
 		replace: map[string]string{
-			"table_option":                 "table_name",
+
```

Comment:
This whole file looks like gofmt messed something up. I'm going to wait on this being fixed and the newlines going away before reviewing it.

Reply:
This was discussed earlier offline -- this is a discrepancy due to me using go 1.11 to format and the gofmt in go 1.10 being broken for this particular syntax construct. Adding the new lines avoids the bug in the 1.10 gofmt.

Reply:
Adding the newlines also makes this file really hard to review because I can't see if any lines actually changed or if it's just whitespace.

Reply:
ok fair. I have manually reverted the problematic lines. RFAL.